itests/vector_test.go

```diff
@@ -30,8 +32,8 @@ func TestCalibrationVector(t *testing.T) {
 		t.Run(filepath.Base(vf.File.Name()), func(t *testing.T) {
 			tvb := NewVectorWalkValidatorBuilder(vf).
 				WithDatabase(strg).
-				WithRange(vf.From, vf.To-1). // TODO file bug
```

Comment:
we can now walk the entire to and from range with no weird off-by-one errors, yay.